sdk/servicebus/Azure.ResourceManager.ServiceBus/src/autorest.md

```diff
@@ -114,6 +114,21 @@ rename-mapping:
   CheckNameAvailabilityResult: ServiceBusNameAvailabilityResult
   NetworkRuleSet: ServiceBusNetworkRuleSet
   NetworkRuleSet.properties.trustedServiceAccessEnabled: IsTrustedServiceAccessEnabled
+  NamespaceReplicaLocation: ServiceBusNamespaceReplicaLocation
+  NetworkSecurityPerimeter: ServiceBusNetworkSecurityPerimeter
+  NetworkSecurityPerimeterConfiguration: ServiceBusNetworkSecurityPerimeterConfiguration
+  NetworkSecurityPerimeterConfigurationPropertiesProfile: ServiceBusNetworkSecurityPerimeterConfigurationPropertiesProfile
+  NetworkSecurityPerimeterConfigurationPropertiesResourceAssociation: ServiceBusNetworkSecurityPerimeterConfigurationPropertiesResourceAssociation
+  NetworkSecurityPerimeterConfigurationProvisioningState: ServiceBusNetworkSecurityPerimeterConfigurationProvisioningState
+  NspAccessRule: ServiceBusNspAccessRule
+  NspAccessRuleDirection: ServiceBusNspAccessRuleDirection
+  NspAccessRuleProperties: ServiceBusNspAccessRuleProperties
+  ResourceAssociationAccessMode: ServiceBusResourceAssociationAccessMode
+  Mode: ServiceBusConfidentialComputeMode
+  ProvisioningIssue: ServiceBusNSPConfigurationProvisioningIssue
+  ProvisioningIssueProperties: ServiceBusNSPConfigurationProvisioningIssueProperties
```

Comment:
looks like we have an inconsistency on the abbr `NSP`, could we align them all to be `Nsp` instead?